### ISSUE TITLE
Varia: Comment reply title skips heading hierarchy

### ISSUE THREAD

**Issue by AtrumGeost** [Automattic/themes#2218]
## Steps to replicate

Create a new blog post and check the comment reply title at the end of the blog post. Then use an accessibility tool to evaluate the post:

![Screen Shot on 2020-07-02 at 21:36:30](https://user-images.githubusercontent.com/3696121/86425972-25a21f00-bcac-11ea-8540-37e8c1be1869.png)

## Result

The comment reply title is using an H3 heading level.

## Expected

The comment reply title should use an H2 heading level.

* Dogfooding
* Slack convo: p1593446135037700-slack-C0FGK24F9
